### ISSUE TITLE
unbork the email

### ISSUE THREAD

**Issue by markdav-is** [MVPStudio/main#32]
Looks like we changed the DNS for mvpstudio.org and it's borked the email settings.

![image](https://user-images.githubusercontent.com/311063/53754971-4402cb00-3e6a-11e9-9755-74c7b80b1308.png)

**Comment by markdav-is** [comment]
from the slack:

scott lively [6 minutes ago]
@markdav.is The domain is still managed by google domains, the DNS is managed by google cloud dns in our google cloud account, and the DNS entry points to the kubernetes cluster.


scott lively [6 minutes ago]
That update was part of making this happen: https://eugenetech.slack.com/archives/GF4G2UYMB/p1551342624000400
scott lively
mvpstudio.org and www.mvpstudio.org are live on kubernetes
Posted in #mvp-studio-devFeb 28th at 12:30 AMView message


Mark Davis (mod) [3 minutes ago]
gotcha.  so that broke the email.  not that anyone was using the email.  I'm not super smart about rebuilding the synthetic records for MX.  I guess the options are restore the google domains server and then point the root and www to whatever K8 needs or re-create the MX situation that would provide the email we had previously


Mark Davis (mod) [2 minutes ago]
previous email mappings
Pasted image at 2019-03-04, 11:15 AM



Mark Davis (mod) [2 minutes ago]
let's take a look at this tuesday!

**Comment by markdav-is** [comment]
![image](https://user-images.githubusercontent.com/311063/53756874-1f5d2200-3e6f-11e9-8833-51afdf03b69e.png)

**Comment by oliverdain** [comment]
Sounds like maybe we just need an MX record?

**Comment by slively** [comment]
There is an mx record, but google domains did some special email forwarding for us.

**Comment by markdav-is** [comment]
let's just g-suite this thing

**Comment by slively** [comment]
Since @oliverdain has the most experience I'm gonna vote he set it up at his convenience. Also because we need it to tie to our gcloud account somehow.

**Comment by oliverdain** [comment]
Will try to get it going this weekend. Who wants to pay the $5 to have a "real" email?

**Comment by slively** [comment]
I do

**Comment by oliverdain** [comment]
Working on this now...

@slively you want scott@mvpstudio.. or slively@ or ??

**Comment by oliverdain** [comment]
For the record (in case I screw something up) here's the steps I took:

1. Add GSuite as an optional add-on to the account
2. Added a GSuite Users sub-organizational unit so we can easily manage all the gsuite folks together
3. For each user who wants an email click on that user and then click on "licenses" at the bottom of the info for that user. Add a GSuite license for the user.

I also had to change our MX records in gcloud. They *were* set to:

```
5 gmr-smtp-in.l.google.com.
10 alt1.gmr-smtp-in.l.google.com.
20 alt2.gmr-smtp-in.l.google.com.
30 alt3.gmr-smtp-in.l.google.com.
40 alt4.gmr-smtp-in.l.google.com.
```

Per [Google instructions](https://admin.google.com/AdminHome?hl=en#SetupWizard/setuptask=OTHER_MX_INSTRUCTIONS_SUBTASK) I changed them to:

```
1 ASPMX.L.GOOGLE.COM.
5 ALT1.ASPMX.L.GOOGLE.COM.
5 ALT2.ASPMX.L.GOOGLE.COM.
10 ALT3.ASPMX.L.GOOGLE.COM.
10 ALT4.ASPMX.L.GOOGLE.COM.
```

That alone didn't work. For some reason the existing MX record was for `@.mvpstudio.org.` rather than `mvpstudio.org.`. Adding a new record for `mvpstudio.org.` did the trick. I deleted the old `@` prefixed one.

For users who don't want a gsuite account we can add forwarding rules for them by logging into the admin console (admin.google.com) and then:

Apps -> GSuite -> Gmail and click on "default routing". That then lets you set catchall routing, forwarding for individual emails, etc.
